__tests__/components/ui/SideNavigation/_partials/Action.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { ActivableLink } from "@/components/ui/SideNavigation/_partials/Action";
+import "@testing-library/jest-dom";
+import { fireEvent, render, screen } from "@testing-library/react";
+
+describe("Activable link", () => {
+  const Emptyitem = {
+    subitems: [],
+    link: "/example",
+  };
+
+  const items = {
+    subitems: [{ label: "subitem", link: "subitem link" }],
+    link: "/dummy",
+  };
+  it("should render the link with subitems", () => {
+    render(<ActivableLink item={items}>Link test</ActivableLink>);
+    const link = screen.getByTestId("activableLinkId");
+    expect(link).toBeInTheDocument();
+    expect(link.textContent).toBe("Link test");
+  });
+
+  it("should render the link with no subitems", () => {
+    render(<ActivableLink item={Emptyitem}>Link test</ActivableLink>);
+    const link = screen.getByText("Link test");
+    expect(link.tagName).toBe("A");
+    expect(link).toHaveAttribute("href", Emptyitem.link);
+  });
```

Comment:
You already tested the items, I don't think you need to test the empty items case